src/issue/screens/new-issue.screen.js

```diff
@@ -90,17 +92,19 @@ class NewIssue extends Component {
       ).then(issue => {
         navigation.navigate('Issue', {
           issue,
+          headerLeft: false,
```

Comment:
With `navigation.reset`, we can't navigate back ✅  
**BUT** the bottom menu item selected when we create the issue can no longer be used 😞
It redirect back to the created issue instead of its normal behavior...